DigitalLearningSolutions.Web.Tests/ViewModels/LearningMenu/IntitialMenuViewModelTests.cs

```diff
@@ -145,10 +145,11 @@ public void Initial_menu_should_have_list_of_section_card_view_model()
             const string sectionName = "TestSectionName";
             const bool hasLearning = true;
             const double percentComplete = 12.00;
+            const int customisationId = 1;
             var expectedCourseContent = CourseContentHelper.CreateDefaultCourseContent();

```

Comment:
Not due to this change but this should not have a name prefixed by `expected`. As we're not using this as an expected value, so just `courseContent` would be a better name 🙂